test/packaging/ansible/roles/agent-dockerized/tasks/docker-compose-Amazon.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Run docker compose
```

Comment:
maybe we could have used https://docs.ansible.com/ansible/latest/collections/community/docker/docker_compose_module.html ?